theano/sandbox/gpuarray/__init__.py

```diff
@@ -52,11 +56,21 @@ def init_dev(dev):
             optdb.add_tags('gpuarray_opt', 'fast_run', 'fast_compile')
         elif (config.init_gpu_device.startswith('cuda') or
               config.init_gpu_device.startswith('opencl')):
+            if config.device != 'gpu':
```

Comment:
It's the same as https://github.com/Theano/Theano/blob/master/theano/sandbox/cuda/__init__.py#L539